docs/til/Sources.md

```diff
@@ -210,7 +210,10 @@ Secret class: [aws](Secret-References.md#aws-secret-class)
 
 ```hcl
 source azure_activity_logs "audit_logs" {
-  event_hub_id = "/subscriptions/1234/resourceGroups/my-group/providers/Microsoft.EventHub/namespaces/mylogs"
+  subscription_id = "1234"
+
+  event_hubs_namespace_id = "/subscriptions/1234/resourceGroups/my-group/providers/Microsoft.EventHub/namespaces/mylogs"
```

Comment:
is it really how it works ? I am asking because it seems that we could extract the subscription id from the event hubs namespace id string.

Reply:
What you mentioned is how it _used to work_, but that API was

- unintuitive
- limiting (the Event Hub isn't necessarily in the same subscription as the one you want to collect activity logs from)

Reply:
thanks got it